src/CodeGenerator/src/Generator/InputGenerator.php

```diff
@@ -97,6 +97,9 @@ public function generate(Operation $operation): ClassName
         $constructorBody = '';
 
         foreach ($shape->getMembers() as $member) {
+            if ('region' === $member->getName()) {
```

Comment:
Is it `@region` or `region`?

Reply:
the MultiRegion hack use @region. But here I want to prevent Input that overlap setRegion and getRegion